src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,73 @@
+package duke;
+
+import duke.data.DataManager;
+import duke.task.TaskManager;
+import duke.user_interface.UserInterface;
+
+public class Duke {
+
+    private static final String dataFilePath = "data/data.txt";
+
+    private static final UserInterface ui = new UserInterface();
+    private static final TaskManager taskManager = new TaskManager();
+    private static final DataManager dataManager = new DataManager(dataFilePath);
+
+    public static void main(String[] args) {
+
+        ui.showWelcomeMessage();
+
+        taskManager.setData(dataManager.loadData());
+
+        while (true) {
```

Comment:
Avoid varying the level of abstraction within a code fragment. Refer to SLAP in code quality